ansible/group_vars/all

```diff
@@ -48,3 +48,37 @@ db:
   authkeys:
   - guest
   - whisk.system
+
+# The default value for catalog_source is empty, meaning that
+# openwhisk will not install the catalog from the catalog repositories.
+# If catalog_source is not set to "catalog-repos", the other variables
+# like catalog_namespace, catalog_auth_key and catalog_repos will not
+# be effective.
+catalog_source: ""
+
+# The default name space is /whisk.system. The catalog namespace must begin with a slash "/".
+catalog_namespace: "/whisk.system"
+
+# The catalog_auth_key is used to determine the secret key to authenticate the openwhisk service.
+# The value for this variable can be set to either the secret key itself or the file, which
+# saves the secret key.
+# By default, we take the key from {{ openwhisk_home }}/ansible/files/auth.whisk.system.
+catalog_auth_key: "{{ openwhisk_home }}/ansible/files/auth.whisk.system"
+
+# The catalog_repos is used to specify all the catalog names and repository URLs,
+# so that openwhisk knows where to download the catalog and install them. The key
+# specifies the catalog name and the url saves the URL of the repository. The location
+# specifies the location to save the code of the catalog.
+# Too add a new repository, please follow the template by adding:
+#
+# catalog_repos:
+#   ...
+#   <catalog-name>:
+#     url: <URL of repository>
+#     location: <local location to save the catalog>
+#
+catalog_repos:
+  openwhisk-catalog:
+    url: https://github.com/openwhisk/openwhisk-catalog.git
```

Comment:
Do you mean we can expect a commit hash like "89915b4cc0810a9c9e67b3706a2850c58120cf75", so that we can checkout and use?